source/Sylvan.Data.Csv.Tests/CsvDataReaderTests.cs

```diff
@@ -167,7 +167,7 @@ public async Task Quoted()
 				Assert.Equal(1, csv.RowNumber);
 				Assert.Equal("1", csv[0]);
 				Assert.Equal("John", csv[1]);
-				Assert.Equal("Very\r\nLow\r\n", csv[2]);
+				Assert.Equal($"Very{Environment.NewLine}Low{Environment.NewLine}", csv[2]);
```

Comment:
I was a little surprised by this one. Is this because git will change the line-endings in Quote.csv when you pull to linux/macos?

Reply:
I'm not really sure about this. I know that the git [core.autocrlf](https://stackoverflow.com/questions/170961/whats-the-best-crlf-carriage-return-line-feed-handling-strategy-with-git) setting is a big mess and I never took the time to fully understand it. But as soon as I saw the assert failing with no *visible* difference [I knew](https://twitter.com/0xced/status/1337378323798876163) it was about CRLF vs LF.